packages/instant/src/components/zero_ex_instant_container.tsx

```diff
@@ -29,6 +30,13 @@ export class ZeroExInstantContainer extends React.Component<{}, ZeroExInstantCon
     public state = {
         tokenSelectionPanelAnimationState: 'none' as SlideAnimationState,
     };
+    private _handlePanelCloseClickedX: () => void;
+    private _handlePanelCloseAfterChose: () => void;
+    public constructor(props: {}) {
+        super(props);
+        this._handlePanelCloseClickedX = this._handlePanelClose.bind(this, TokenSelectorClosedVia.ClickedX);
+        this._handlePanelCloseAfterChose = this._handlePanelClose.bind(this, TokenSelectorClosedVia.TokenChose);
```

Comment:
can we just make these they're own functions with implementations below? i.e.

```javascript
private readonly _handlePanelCloseClickedX = (): void => {
    this._handlePanelClose(TokenSelectorClosedVia.ClickedX);
};
```

I this is clearer

Reply:
I can do this, but they both would have to call 
```
        this.setState({
            tokenSelectionPanelAnimationState: 'slidOut',
        });
```
or a helper function which does this, which feels a little repetitive.  Would prefer to keep the existing implementation, but can change if you feel strongly